xbmc/pvr/epg/EpgDatabase.cpp

```diff
@@ -87,7 +87,7 @@ void CPVREpgDatabase::CreateTables(void)
         "iFirstAired     integer, "
         "iParentalRating integer, "
         "iStarRating     integer, "
-        "bNotify         bool, "
+        "bNotify         bool, " //! @todo unused. remove with next db schema change.
```

Comment:
Why not remove straight away?

Reply:
I wanted to avoid a db version bump (and migration) just for one removed bool. If you feel this is wrong, I can ofc, change that.

Reply:
I'm afraid no one will see this todo when they do the next bump :P

Reply:
Okay, I will add the db bump.

Reply:
SQLITE does not support "ALTER TABLE x DROP COLUMN y"

"You can not use the ALTER TABLE statement to drop a column in a table. Instead you will need to rename the table, create a new table, and copy the data into the new table."

This is getting nasty.  Mind if I just keep the (now unused column) in the table?